config/staging/daily.json

```diff
@@ -19,6 +14,10 @@
         ],
         "tags": [
             "nightly"
+        ],
+        "trees": [
+            "mozilla-central",
+            "mozilla-aurora"
```

Comment:
Good feedback but actually this was a mistake a long time ago. There are no daily builds for beta, so no need to keep that.